interface/main/calendar/modules/PostCalendar/pntemplates/default/views/header.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html>
 <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-[-headerTemplate-]
+    <meta charset="utf-8" />
+    [-headerTemplate-]
+    <title>[-php-] echo xlt('Calendar'); [-/php-]</title>
```

Comment:
better way to do this in pure smarty is:
```
<title>[-xlt t='Calendar'-]</title>
```

Reply:
lol I just moved the title down one line it was already there

Reply:
Not seeing that.
Are you sure you didn't take it from main_info.php code?
(note this is a minor issue here since the calendar templating is the only place left that breaks smarty templating with pure php; ie. can leave it as it is)